README.md

```diff
@@ -62,6 +62,11 @@ navigationController?.pushViewController(readerController, animated: true)
 PDFViewController.createNew(with: document, title: "Favorite Cupcakes")
 ```
 
+#### Background Color
+```swift
+controller.setBackgroundColor(to: UIColor.white)
```

Comment:
If you're calling a function with a param expecting a UIColor, then you don't need to re-specify this type information. For example, the follow would be fine:

``` swift
controller.setBackgroundColor(to: .white)
```